plugins/logging/CMakeLists.txt

```diff
@@ -32,6 +32,8 @@ include_directories (
          ./../../system/include 
          ./../../system/jlib 
          ./../../rtl/include
+         ./../../rtl/eclrtl
+         ./../../common/thorhelper
```

Comment:
I don't like adding this dependency - (or the dynamic cast).  Logically could this functionality apply to any engine?

Reply:
I had considered that, but wasn't as sure of equivalent meaning in other components.  I suppose basing it on workunit start time would be most equivalent to the roxie start of query?  I would then make the roxie running from workunit queue version based on that as well.